Team417_CENTERSTAGE/src/main/java/org/firstinspires/ftc/team417_CENTERSTAGE/BaseTeleOp.java

```diff
@@ -0,0 +1,249 @@
+package org.firstinspires.ftc.team417_CENTERSTAGE;
+
+import com.qualcomm.robotcore.eventloop.opmode.TeleOp;
+
+@TeleOp(name = "TeleOp League 1")
+public class BaseTeleOp extends BaseOpMode {
+    @Override
+    public void runOpMode() {
+        initializeHardware();
+
+        waitForStart();
+
+        while (opModeIsActive()) {
+            driveUsingControllers(false);
+            outputUsingControllers();
+            intakeUsingControllers();
+            telemetry.addData("FRMotor", FR.getCurrentPosition());
+            telemetry.addData("FRMotor", FR.getPowerFloat());
+            telemetry.addData("FLMotor", FL.getCurrentPosition());
+            telemetry.addData("FLMotor", FL.getPowerFloat());
+            telemetry.addData("BRMotor", BR.getCurrentPosition());
+            telemetry.addData("BRMotor", BR.getPowerFloat());
+            telemetry.addData("BLMotor", BL.getCurrentPosition());
+            telemetry.addData("BLMotor", BL.getPowerFloat());
+            telemetry.addData("ArmMotor", armMotor.getCurrentPosition());
+            telemetry.addData("DumperServo", dumperServo.getPosition());
+            telemetry.update();
+        }
+    }
+
+    public boolean sensitive = false;
+
+    public void driveUsingControllers() {
+        sensitive = gamepad1.right_bumper;
```

Comment:
This routine duplicates a bunch of code of the driveUsingControllers(boolean) routine just below. Any time you find that there's a bunch of code that looks exactly alike, think twice about whether it's needed. In this case, driveUsingControllers() could probably just call driveUsingControllers(false). The duplicate code would be eliminated.